doc/doxygen/references.bib

```diff
@@ -1478,3 +1478,40 @@ @article{kronbichler2019multigrid
   year={2019},
   publisher={ACM New York, NY, USA}
 }
+
+@article{Boffi2011,
+  doi = {10.1007/s10915-011-9549-4},
+  url = {https://doi.org/10.1007/s10915-011-9549-4},
+  year = {2011},
+  month = nov,
+  publisher = {Springer Science and Business Media {LLC}},
+  volume = {52},
+  number = {2},
+  pages = {383--400},
+  author = {D. Boffi and N. Cavallini and F. Gardini and L. Gastaldi},
+  title = {Local Mass Conservation of Stokes Finite Elements},
+  journal = {Journal of Scientific Computing}
+}
+
+@article{taho73,
```

Comment:
Can we rename this into 
```suggestion
@article{Taylor73,
```
to give those people who do not want to click the reference an idea what the paper is about? (I do not like to click on reference links and just look at the name of the web page linked to, which picks up this reference and thus I like expressive names for the papers.)